db/migrate/20191202202212_add_featured_topic_id_to_user_profiles.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+class AddFeaturedTopicIdToUserProfiles < ActiveRecord::Migration[6.0]
+  def change
+    add_reference :user_profiles, :featured_topic, foreign_key: { to_table: 'topics' }
```

Comment:
We've had some debate internally about using foreign keys. While we might use them in the future for now it's part of a bigger discussion we haven't finished.

It would be better if you used a regular NULLable int here